shared/ui/core/src/main/java/com/ivy/ui/FormatMoneyUseCase.kt

```diff
@@ -7,6 +7,11 @@ import dagger.hilt.android.qualifiers.ApplicationContext
 import java.text.DecimalFormat
 import java.text.DecimalFormatSymbols
 import javax.inject.Inject
+import kotlin.math.abs
+
+const val MILLION = 1_000_000
+const val BILLION = 1_000_000_000
+const val THOUSAND = 1_000
```

Comment:
Sort the lines to be THOUSAND, MILLION, BILLION instead of MILLION, BILLION, THOUSAND